plugins/utils/mark-radar.nix

```diff
@@ -0,0 +1,48 @@
+{ pkgs, config, lib, ... }:
+
+with lib;
+let
+  cfg = config.programs.nixvim.plugins.mark-radar;
+  helpers = import ../helpers.nix { inherit lib; };
+  defs = import ../plugin-defs.nix { inherit pkgs; };
+in {
+  options.programs.nixvim.plugins.mark-radar = {
+    enable = mkEnableOption "Enable mark-radar";
+
+    highlight_background = mkOption {
```

Comment:
Sorry for continuing to press on this 😅 but it's best to make these options default to null (type being types.nullOr [original type]), to avoid filling up the init file with unnecessary options. I can take care of this tonight though!

Reply:
No, you're perfectly right! I've pushed a fix. 
Please do keep reviewing if you find something off.